### PR TITLE
Doc: tutorial of fault-tolerance and elasticity experiments.

### LINKED ISSUE #734: A tech report to test the fault tolerance and elasticity using chaosblade.
We can use chaosblade to create chaos experiments:
- Pod is preempted.
- The training process hangs or fails.
- The cpu is busy to test straggler.
- OOM of Pod.
- Pod network is breakdown .

### LINKED ISSUE #735: The elapsed time of network check is not changed.
```
The elapsed time of nodes are {2: 20.182805944442748, 3: 18.725976165771485, 0: 181.80825593948364, 1: 143.06098918533326}
The elapsed time of nodes are {2: 20.182805944442748, 3: 18.725976165771485, 0: 19.91257585334778, 1: 143.06098918533326}
The elapsed time of nodes are {2: 20.16765477180481, 3: 18.725976165771485, 0: 19.91257585334778, 1: 20.607787996292114}
```

### PR DESCRIPTION
**Pull request by workingloong**
### What changes were proposed in this pull request?

A tutorial of fault-tolerance and elasticity experiments.

### Why are the changes needed?

Fix #734 

### Does this PR introduce any user-facing change?

No.

### How was this patch tested?

Just a document.

### CODE REVIEW

File: docs/tutorial/fault_tolerations.md [hunk 1]
```diff
@@ -1,346 +1,494 @@
...
-## worker容错示例
+The tutorial shows experiments to test the fault-tolerance and elasticity
+of DLRover ElasticJob. In the experiments, we use the chaos enginerring toolkit
+[chaosblade](https://github.com/chaosblade-io/chaosblade) to simulate fault scenarios.
 
-在任务运行过程中删除worker-i对应的pod，之后dlrover master会重新拉起一个pod。work-i对应的pod的名称会发生变化，新创建的pod的启动命令和被kill掉的pod的启动命令相同，启动后参与组网，并进行训练。期间其他worker不受影响。
+## Preliminary
 
-### 启动作业
+- Create a k8s cluster and configure cluster credentials on your local computer.
```
Comment (merlintang) [inline docs/tutorial/fault_tolerations.md:9]: why we need to configure the cluster credentials on the local machine?

File: docs/tutorial/fault_tolerations.md [hunk 2]
```diff
@@ -1,346 +1,494 @@
...
+In the experiment, we set replicas of worker to 4 in a job and
+use chaosblade to perform a CPU full load 90% on the `worker-1` with the command
+
+```bash
+chaosblade-1.7.2/blade create cpu load --cpu-percent 90
 ```
 
-## PS 容错示例
+If you use the image `registry.cn-hangzhou.aliyuncs.com/intell-ai/dlrover:torch201-mnist`,
+you can use chaosblade to create an chaos experint by
```
Comment (merlintang) [inline docs/tutorial/fault_tolerations.md:92]: experint means?

File: docs/tutorial/fault_tolerations.md [hunk 3]
```diff
@@ -1,346 +1,494 @@
...
+
+```text
+Round 1: The node status are {1: False, 2: True, 3: True, 0: False}.
+Round 2: The node status are {1: False, 2: True, 3: True, 0: True}.
+Round 3: The node status are {3: True, 0: True, 1: True, 2: True}.
+```
+
+### Network Breakdown
+
+In the experiment, we set replicas of worker to 4 in a job and
```
Comment (merlintang) [inline docs/tutorial/fault_tolerations.md:210]: In this experimental study

**Review by merlintang — COMMENTED** [review]
can we have a diagram to illustrate the process when running different situtation. 

maybe Tingfen and others can help on this?

File: docs/tutorial/fault_tolerations.md (hunk 1 above)
Comment (workingloong) [reply docs/tutorial/fault_tolerations.md:9]: The  k8s cluster is usually a remote cluster and we need the credential to connect the cluster.

**Comment by workingloong** [comment]
> can we have a diagram to illustrate the process when running different situtation.

Not yeah. Good suggestion to draw a diagram to illustrate the process of fault-tolerance.

**Review by merlintang — APPROVED** [review]

**Pull request merged**